docs/source/install/upgrades.rst

```diff
@@ -136,8 +136,12 @@ This is the standard upgrade procedure:
 
       sudo yum update st2 st2web st2chatops st2mistral
 
+   Omit st2mistral if it is not installed on your distribution.
```

Comment:
These two comments could use a bit of expansion. Why would st2mistral not be installed on a server? It's required by StackStorm <= 3.2, so it should be there. For StackStorm >= 3.3, it won't be available. We should explain that here, possibly include a link to the changelog entry.

Reply:
@blag it isn’t installed though on EL8 with ST 3.2 though. That was what I was trying to get across.
So I was thinking of it as part of a upgrade on el8 with what we have now.